gpu_bdb/queries/q04/gpu_bdb_query_04_dask_sql.py

```diff
@@ -84,7 +87,11 @@ def main(data_dir, client, c, config):
     result = result.persist()
 
     result = result.compute()
-    result_df = cudf.DataFrame({"sum(pagecount)/count(*)": [result]})
+    
+    if isinstance(merged_df, cudf.DataFrame):
```

Comment:
merged_df is a dask_cudf.DataFrame no , so below check is invalid.   

I would suggest testing with both `backed=GPU/CPU` both so that we dont run into issues as were pointed out in PR https://github.com/rapidsai/gpu-bdb/pull/244